client/landing/gutenboarding/available-designs.ts

```diff
@@ -56,28 +56,16 @@ export function prefetchDesignThumbs() {
 }
 
 export function getAvailableDesigns(
-	includeEdgeDesigns: boolean = isEnabled( 'gutenboarding/edge-templates' ),
 	useFseDesigns: boolean = isEnabled( 'gutenboarding/site-editor' )
 ) {
-	let designs = availableDesigns;
-
-	if ( ! includeEdgeDesigns ) {
-		designs = {
-			...designs,
-			featured: designs.featured.filter( ( design ) => ! design.is_alpha ),
-		};
-	}
-
 	// If we are in the FSE flow, only show FSE designs. In normal flows, remove
 	// the FSE designs.
-	designs = {
-		...designs,
-		featured: designs.featured.filter( ( design ) =>
+	return {
+		...availableDesigns,
```

Comment:
@noahtallen Making sure this doesn't break your stuff :)

Reply:
I think this would break it 😅 We used that alpha section to make sure the FSE designs only show up on horizon/wpcalypso. Why not keep it? it seems really useful

Reply:
> Why not keep it? it seems really useful

_Reverting..._ ✅

Reply:
haha thanks! 🙌